### ISSUE TITLE
Very noisy logging

### ISSUE THREAD

**Issue by johtso** [bibanon/tubeup#193]
I haven't managed to pinpoint what line is responsible, but currently when uploading a very large object is printed to the terminal:

![image](https://user-images.githubusercontent.com/830800/162096298-55967230-6c5f-4219-b044-694c0d5db4e8.png)

**Comment by vxbinaca** [comment]
This started when downloaders were switched. Lemme ask puka about this.

**Comment by brandongalbraith** [comment]
https://github.com/bibanon/tubeup/pull/176 should've fixed this.

**Comment by johtso** [comment]
Seems to still be doing it on current master.

**Comment by brandongalbraith** [comment]
![image](https://user-images.githubusercontent.com/350893/162097757-334bfd7f-9aea-42dc-98e3-5429346b9a32.png)

We never cut a new release 😱

**Comment by johtso** [comment]
@brandongalbraith I've tried the master branch which includes that commit and the object was still being logged.

**Comment by brandongalbraith** [comment]
Thank @johtso, will have to do some digging this evening then to see why the verbosity is still happening after flipping the flag for it.

**Comment by brandongalbraith** [comment]
`forcejson = False` [didn't resolve this](https://github.com/bibanon/tubeup/pull/176#issuecomment-1100673246) as expected, more research required on my part on how to suppress this verbose output.

**Comment by vxbinaca** [comment]
I will say recently, the video title is printed before the JSON to STDOUT. I personally like that the title is printed, but thats just me. Could be a YT-DLP thing.

**Comment by pukkandan** [comment]
The object is being dumped by this line: https://github.com/bibanon/tubeup/blob/84bc1fc46f859385a437e96909871c07d9551770/tubeup/TubeUp.py#L139

**Comment by vxbinaca** [comment]
@pukkandan this started when we switched downloaders if it's any help, but switching downloaders slayed like 5 other much more big dragons. So in balance I'd take it. Although I'm not opposed to fixing this of course.

**Comment by mrpapersonic** [comment]
> @pukkandan Thanks for surfacing that, going to do some testing to suppress dumping that object unless `verbose = True`.

The `verbose` flag is a bit misleading; in relation with __main__.py, it really just means "not quiet". In fact, the code already checks for `verbose`.

All that's needed to fix this is commenting out the `print` call [here](https://github.com/bibanon/tubeup/blob/master/tubeup/TubeUp.py#L155) and possibly the `self.logger.debug` call as well.

**Comment by vxbinaca** [comment]
> > @pukkandan Thanks for surfacing that, going to do some testing to suppress dumping that object unless `verbose = True`.
> 
> The `verbose` flag is a bit misleading; in relation with __main__.py, it really just means "not quiet". In fact, the code already checks for `verbose`.
> 
> All that's needed to fix this is commenting out the `print` call [here](https://github.com/bibanon/tubeup/blob/master/tubeup/TubeUp.py#L155) and possibly the `self.logger.debug` call as well.
> 
> TL;DR: `verbose` will always be set to true unless --quiet is passed.

Couldn't I just delete that line instead of commenting it out?

**Comment by vxbinaca** [comment]
done, lemme pull the result and see if it's fixed, I'll try the second one if the first one worked, unit tests weren't complaining

**Comment by vxbinaca** [comment]
It is for sure squashed thank you for pointing this out to me it's been open so long it's like ambient noise you don't hear it anymore.
